EIPS/eip-2681.md

```diff
@@ -25,7 +25,7 @@ Lastly, this facilitates a minor optimisation in clients, because the nonce no l
 
 Introduce two new restrictions retroactively from genesis:
 
-1. Consider any transaction invalid, where the nonce exceeds `2^64-1`.
+1. Consider any transaction invalid, where the nonce exceeds or equals to `2^64-1`.
```

Comment:
This is a normative change, which makes me fairly uncomfortable making it as an edit to a final EIP.  One could create a new EIP that adds 2^64-1 to the set of allowed nonces, but policy is that final EIPs can have non-normative changes only.